Jenkinsfile

```diff
@@ -77,7 +77,7 @@ try {
     // If we're on master and it failed, notify slack
     if (env.BRANCH_NAME == "master") {
         def currentResult = currentBuild.result ?: 'SUCCESS'
-        def channel = '#team-dropbear-priv'
+        def channel = '#priv-pegasys-prod-dev'
```

Comment:
why not dropBear chat?Or do we want everyone to know about this? (I spose that's good if we ever don't look after this product...)

Reply:
I was thinking that we are finishing up with ethSigner so would be better for everyone to be able to get notifications if they working on it.